etc/telegraf.conf

```diff
@@ -784,13 +784,18 @@
 #   ## Timeout for HTTP requests to the elastic search server(s)
 #   http_timeout = "5s"
 #
-#   ## set local to false when you want to read the indices stats from all nodes
-#   ## within the cluster
+#   ## When local is true (the default), the node will read only its own stats.
+#   ## Set local to false when you want to read the node stats from all nodes
+#   ## of the cluster.
 #   local = true
 #
-#   ## set cluster_health to true when you want to also obtain cluster level stats
+#   ## set cluster_health to true when you want to also obtain cluster health stats
 #   cluster_health = false
 #
+#   ## Set cluster_stats to true when you want to obtain cluster stats from the
+#   ## Master node. This is recommended for use only when local = true
```

Comment:
I don't know enough about elasticsearch to say one way or the other definitively. If you think it makes sense then please make it consistent.

Also could you add more explanation as to why you make this recommendation? it's not entirely clear from the documentation.